.github/actions/package-builder/action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Package Builder'
+description: 'Upload version and build package. Upload build files'
```

Comment:
We can add more detail as: "Creates a and uploads a build artifact updating the version of the package and its dependencies"